### PR TITLE
gh-36 adding manual reheating control

### PR DESCRIPTION
**Pull request by dfreeman06**
## References

gh-36 manual reheating.

## Code changes

Using a custom message to tell the ForceGraphView to reheat the `ForceGraphInstance`. Both the 2d and 3d have a `d3ReheatSimulation` method.

## User-facing changes

Added an example ui button for reheating in the `Forces` notebook.

## Backwards-incompatible changes

None expected.

### CODE REVIEW

File: js/widgets/display/2d.ts [hunk 1]
```diff
@@ -229,11 +230,21 @@ export class ForceGraphView<T = ForceGraphGenericInstance<ForceGraphInstance>>
...
     this.luminoWidget.disposed.connect(this.onDisposed, this);
-
+    this.model.on('msg:custom', this.handleMessage, this);
     this.onSourceChange();
     this.onBehaviorsChange();
   }
 
+  handleMessage(content: IActionMessage): void {
+    switch (content.action) {
+      case 'reheat':
```
[the remark this answers is not in the document]
Comment (nrbgt) [reply js/widgets/display/2d.ts:240]: I'll take a whack at this as a sub-PR to try the :badger:

[the remark this answers is not in the document]
Comment (nrbgt) [reply js/widgets/display/2d.ts:240]: #55 ready for review, would then happily approve this one

**Review by sanbales — APPROVED** [review]
Had already reviewed these in Nick's PR, but just did a quick pass and everything seems kosher.

**Comment by nrbgt** [comment]
I'm going to skip the failing osx tests

**Pull request merged**